package.json

```diff
@@ -109,6 +111,7 @@
     "eslint-plugin-jsx-a11y": "6.6.1",
     "eslint-plugin-n": "15.6.0",
     "eslint-plugin-node": "11.1.0",
+    "eslint-plugin-playwright": "^0.15.3",
```

Comment:
Can you please pin the version number by running `yarn add -D eslint-plugin-playwright@0.15.3`?